### PR TITLE
changing deployment name

### PR DESCRIPTION
**Pull request by shrutir25**
**What this PR does / why we need it**: Changes the way deployment name is computed

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
Currently ARM deployment names do not have a cap for 64 characters. Changing the way this name is computed. Making it a function of time/suffix.

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by shrutir25** [comment]
@dmitsh ptal

File: test/e2e/dcos/dcos.go [hunk 1]
```diff
@@ -235,6 +235,9 @@ func (c *Cluster) AppCount() (int, error) {
 // Version will return the node count for a dcos cluster
 func (c *Cluster) Version() (string, error) {
 	out, err := c.Connection.Execute("curl -s http://localhost:80/dcos-metadata/dcos-version.json")
+	if err != nil {
```
Comment (jackfrancis) [inline test/e2e/dcos/dcos.go]: Is this code related?

Comment (dmitsh) [reply test/e2e/dcos/dcos.go]: we were getting lint warning ...

Comment (jackfrancis) [reply test/e2e/dcos/dcos.go]: Nah, it's fine, strange that it should choose this PR to complain about that!

Comment (shrutir25) [reply test/e2e/dcos/dcos.go]: So my CI build failed with make test-style. And I fixed all the things it complained about. This showed up as a warning. And my guess is that it has been ignored all this while. Also that check is needed. We are calculating err and not using it.
